src/lib/utils/translations.ts

```diff
@@ -1,9 +1,45 @@
-import i18nConfigs from "../../../i18n.config.json"
+import i18nConfig from "../../../i18n.config.json"
 import { DEFAULT_LOCALE } from "../constants"
-import { Lang } from "../types"
+import { I18nLocale, Lang } from "../types"
+
+export type Languages = {
+  [lang in Lang]: I18nLocale
+}
+
+export const defaultLanguage: Lang = "en"
+
+// same data as in the `config.json` but indexed by language code
+const languages: Languages = i18nConfig.reduce((result, config) => {
+  return { ...result, [config.code]: config }
+}, {} as Languages)
+
+const buildLangs = (process.env.GATSBY_BUILD_LANGS || "")
+  .split(",")
+  .filter(Boolean)
+
+// will take the same shape as `languages`. Only thing we are doing
+// here is filtering the desired langs to be built
+export const languageMetadata: {
```

Comment:
@nhsz I see that we are just importing the `languageMetadata` in the languages page.

Most of the things defined here are not used anymore since those things are handled now in the `next-i18next.config.js` file.

Or am I missing something?

I think we could get rid of: 
- `buildLangs`
- `languageMetadata` (and just export and use the `languages` object)
- `supportedLanguages`
- `ignoreLanguages`

LMK what do you think

Reply:
Fixed